kyuubi-server/web-ui/src/App.vue

```diff
@@ -0,0 +1,20 @@
+/* * Licensed to the Apache Software Foundation (ASF) under one or more *
+contributor license agreements. See the NOTICE file distributed with * this work
+for additional information regarding copyright ownership. * The ASF licenses
+this file to You under the Apache License, Version 2.0 * (the "License"); you
+may not use this file except in compliance with * the License. You may obtain a
+copy of the License at * * http://www.apache.org/licenses/LICENSE-2.0 * * Unless
+required by applicable law or agreed to in writing, software * distributed under
+the License is distributed on an "AS IS" BASIS, * WITHOUT WARRANTIES OR
+CONDITIONS OF ANY KIND, either express or implied. * See the License for the
+specific language governing permissions and * limitations under the License. */
```

Comment:
this licence header is broken

Reply:
emmm.. It's caused by prettier / eslint, I am not sure if it's broken...

Reply:
I found a positive example.
https://github.com/apache/skywalking-booster-ui/blob/main/src/App.vue

Reply:
yes, nice example. Vue support <!--  --> and /* */ as comment format, <!--  -->  will not formatted by eslint, will change.